src/operator/custom/custom-inl.h

```diff
@@ -143,8 +149,7 @@ class CustomOperator {
 
             ctx.async_on_complete();
           },
-          ctx.run_ctx.ctx, vars, vars2, FnProperty::kNoSkip, 0,
-          "CustomOperator");
+          ctx.run_ctx.ctx, vars, vars2, FnProperty::kNoSkip, 0, "Dummy_Wait");
```

Comment:
Can we change Dummy_Wait -> CustomOperatorWait

Reply:
Sure!